code/game/objects/effects/anomalies.dm

```diff
@@ -147,6 +147,7 @@
 	var/canshock = FALSE
 	var/shockdamage = 20
 	var/explosive = TRUE
+	var/is_zk = FALSE
```

Comment:
Please document this variable and give it a more informative name.
What the hell is zk?